client/ayon_core/tools/tray/addons_manager.py

```diff
@@ -0,0 +1,247 @@
+import os
+import time
+from typing import Callable
+
+from ayon_core.addon import AddonsManager, ITrayAddon, ITrayService
+from ayon_core.tools.tray.webserver import (
+    find_free_port,
+    WebServerManager,
+)
+
+
+class TrayAddonsManager(AddonsManager):
+    # TODO do not use env variable
+    webserver_url_env = "AYON_WEBSERVER_URL"
+    # Define order of addons in menu
+    # TODO find better way how to define order
+    addons_menu_order = (
+        "ftrack",
+        "kitsu",
+        "launcher_tool",
+        "clockify",
+    )
+
+    def __init__(self, tray_manager):
+        super().__init__(initialize=False)
+
+        self._tray_manager = tray_manager
+
+        self._webserver_manager = WebServerManager(find_free_port(), None)
```

Comment:
Shouldn't this somehow pass it the WEBSERVER URL that's configured (e.g. through the env var?)

Reply:
Not sure I understand?

Reply:
Correct - but none of those seem passed to the `WebServerManager`?

Reply:
Because `WebServerManager` is source of the value...